### PR TITLE
update references to destinations to make it clear its not just Kubernetes

### PR DESCRIPTION
**Pull request by aclevername**

### CODE REVIEW

File: docs/main/05-reference/01-deployment-topology/01-deployment-topology.md [hunk 1]
```diff
@@ -4,56 +4,99 @@ title: Deployment topology
 id: intro
```
Comment (aclevername) [inline docs/main/05-reference/01-deployment-topology/01-deployment-topology.md:4]: I just formatted this file, no changes

File: docs/main/05-reference/03-destinations/01-destinations.md [hunk 2]
```diff
@@ -5,10 +5,27 @@ sidebar_label: Destinations
 id: intro
 ---
 
-The Kratix Destination Custom Resource Definition (CRD) is the representation of a system where workloads
-can be scheduled to. These can be Kubernetes clusters or any other infrastructure that can be deployed to using GitOps.
-See below for the API documentation:
-
+The Kratix Destination Custom Resource Definition (CRD) is the representation of
+a system where Kratix can write documents to. These documents are then
```
Comment (SaphMB) [inline docs/main/05-reference/03-destinations/01-destinations.md:9]:
```suggestion
a system that Kratix can write documents to. These documents are then
```

File: docs/main/05-reference/03-destinations/01-destinations.md [hunk 3]
```diff
@@ -5,10 +5,27 @@ sidebar_label: Destinations
...
+reconciled by an external tool.
+
+Some example use cases:
+- [Kubernetes cluster](https://kubernetes.io/): Kratix will scheduled documents (Kubernets manifests) to
+  the Destination, and then a GitOps tool running on the Kubernetes cluster,
+  such as Flux or ArgoCD with pull down the documents and deploy them.
+- [Terraform](https://www.terraform.io/): There are many toolings that exists to trigger Terraform applys
```
Comment (SaphMB) [inline docs/main/05-reference/03-destinations/01-destinations.md:16]:
```suggestion
- [Terraform](https://www.terraform.io/): There are many toolings that exist to trigger Terraform applys
```

File: docs/main/05-reference/03-destinations/01-destinations.md [hunk 4]
```diff
@@ -5,10 +5,27 @@ sidebar_label: Destinations
...
+  when a new terraform file is committed to a Git repository. For example
```
Comment (SaphMB) [inline docs/main/05-reference/03-destinations/01-destinations.md:17]:
```suggestion
  when a new Terraform file is committed to a Git repository. For example
```

**Review by SaphMB — DISMISSED** [review]
Nice and clear, just had a few editing tweaks

**Review by SaphMB — APPROVED** [review]

**Pull request merged**